connectors/slack/template/index.js

```diff
@@ -20,307 +17,222 @@ module.exports = (ctx, cb) => {
 
   // Most requests go to the Settings Manager
   if (!pathname.startsWith('/events')) {
-    Sdk.createSettingsManager(configure)(ctx).then(
-      function (result) {
-        cb(null, result);
-      },
-      function (err) {
-        cb(err, null);
-      }
-    );
+    return Sdk.createSettingsManager(configure)(ctx);
```

Comment:
Should this be an await? (Full disclosure - I can’t see the source code from here to check myself)

Reply:
No, that's consistent with how we do it in all addons